examples/bri-3/src/bri/identity/bpiSubjectAccounts/agents/bpiSubjectAccounts.agent.ts

```diff
@@ -2,6 +2,7 @@ import { Injectable, NotFoundException } from '@nestjs/common';
 import { v4 } from 'uuid';
 
 import { NOT_FOUND_ERR_MESSAGE } from '../api/err.messages';
+import { NOT_FOUND_ERR_MESSAGE as SUBJECT_NOT_FOUND_ERR_MESSAGE } from '../../bpiSubjects/api/err.messages';
```

Comment:
import does not need 'as'

Reply:
yes it needs, they are not the same messages, gonna need to check how this was working at the first place, test expected this message before